packages/tpetra/kernels/perf_test/spmv/test_crsmatrix.cpp

```diff
@@ -61,8 +63,11 @@
 #include <Kokkos_SPMV_Inspector.hpp>
 #include <CuSparse_SPMV.hpp>
 #include <MKL_SPMV.hpp>
+#include <OpenMPStatic_SPMV.hpp>
```

Comment:
These includes currently won't build if OpenMP is disabled.  Please protect the includes with `#ifdef KOKKOS_HAVE_OPENMP ... #endif`.